caf-audit-service-container/README.md

```diff
@@ -7,19 +7,25 @@ This is a docker container for the [CAF Audit Web Service](https://github.com/CA
 #### Environment Variables
 
 ##### CAF\_ELASTIC\_HOST\_AND\_PORT
-Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200
+Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200. Default value is elastic search host value constructed from alternate configurations CAF_ELASTIC_HOST_VALUES and CAF_ELASTIC_PORT_VALUE.
+
+##### CAF\_ELASTIC\_HOST\_VALUES
+This is the alternative configuration with comma seperated list of Elasticsearch host names. eg. localhost Default value: localhost.
+
+##### CAF\_ELASTIC\_PORT\_VALUE
+This is the alternative configuration for REST port of the Elasticsearch server listens on. e.g 9200. Default value: 9200.
 
 ##### CAF\_ELASTIC\_CLUSTER\_NAME
-Name of the Elasticsearch cluster. e.g. docker-cluster
+Name of the Elasticsearch cluster. e.g. docker-cluster. Default value: elasticsearch-cluster.
```

Comment:
Could you confirm this default value for the cluster name?

Reply:
@thilagasanthosh Start the container and confirm these values before sending for review.